subworkflows/local/standardisation_profiles.nf

```diff
@@ -45,8 +84,10 @@ workflow STANDARDISATION_PROFILES {
             bracken: it[0]['tool'] == 'bracken'
             centrifuge: it[0]['tool'] == 'centrifuge'
             kraken2: it[0]['tool'] == 'kraken2'
-            metaphlan3: it[0]['tool'] == 'metaphlan3'
+            metaphlan: it[0]['tool'] == 'metaphlan'
             motus: it[0]['tool'] == 'motus'
+            kmcp: it [0]['tool'] == 'kmcp'
+            ganon: it[0]['tool'] == 'ganon'
```

Comment:
alphabetical order please :pray:

Reply:
Fixed in https://github.com/nf-core/taxprofiler/pull/370